components/desktop/dmenu/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2016 Sergey Avseyev
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		dmenu
+COMPONENT_VERSION=	4.6
+COMPONENT_PROJECT_URL=	http://tools.suckless.org/dmenu/
+COMPONENT_SUMMARY=	dmenu is a dynamic menu for X, originally designed for dwm. It manages large numbers of user-defined menu items efficiently
```

Comment:
can we preserve COMPONENT_SUMMARY short and move longer description to pkg.description if necessary?